src/input/capability.rs

```diff
@@ -63,6 +63,12 @@ impl FromStr for Capability {
             "DBus" => Ok(Capability::DBus(Action::from_str(
                 parts.join(":").as_str(),
             )?)),
+            "Touchpad" => Ok(Capability::Touchpad(Touchpad::from_str(
+                parts.join(":").as_str(),
```

Comment:
clippy will complain about using a string here, use `'` instead of `"` for char.

Reply:
I get errors if I try to do that:
![image](https://github.com/ShadowBlip/InputPlumber/assets/376460/c6f33472-5bbd-467d-a362-c727719ec48d)